CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 1.0.3-dev
+## 1.1.0
```

Comment:
Looks like we rev'd to a non-dev version here, but haven't published `1.1.0` yet? We should likely publish this (or, switch back to `-dev`).